fvm_actor_utils/src/util.rs

```diff
@@ -27,25 +28,39 @@ type ActorResult<T> = std::result::Result<T, ActorError>;
 /// It provides higher level utilities than raw syscalls for actors to use to interact with the
 /// IPLD layer and the FVM runtime (e.g. messaging other actors)
 #[derive(Clone, Debug)]
-pub struct ActorRuntime<S: Syscalls, BS: Blockstore> {
+pub struct ActorRuntime<S: Syscalls + Clone, BS: Blockstore + Clone> {
```

Comment:
While the Syscalls and Blockstore implementations should be made clone-able so that actors and unit tests etc. can share them between `Token` handles, I'm not sure that the ActorRuntime needs to require them to be the case in the trait bounds. 

I may be missing some of the use case, but as far as I can tell, nothing within the ActorRuntime actually uses the fact that the blockstore is cloneable.

Reply:
Without the `Clone` trait bound, you can't actually call `clone()` on the `ActorRuntime` (it knows about cloning itself but as far as it understands, it's composed of components that can't clone). Changing the `Token` to use a reference instead seems to be working well so I've taken out all the clone stuff to use that instead.